### PR TITLE
feat(CenterPoint): release CenterPoint 1.6

### PR DESCRIPTION
**Pull request by KSeangTan**
## Summary
Release CenterPoint v1.6 with the following changes:
 - This release add more training data to `db_j6_v3`
 - Update `db_j6gen2_v1` with the new data `db_j6gen2_v2`
 - It also introduces new data for `db_largebus_v1`
 - It updates number of points per pillar from `20` to `32` to align with the number in `autoware.universe`
 - It further stabilize AMP training by:
 - Introduces `AMPGaussianFocalLoss` to prevent underflow addition in FP16 for `1e-12`
     - Reduce `grad_clip` from `35` to `15`
     - Adjust `init_scale` and `growth_interval` for `loss_scaler`
     - Adjust init values for hetmap bias to `-4.595`
  - Introduces `LossScaleInfoHook` to monitor `loss_scaler`
  - Enable `SafeMLflowVisBackend` for support MLflow 

## Performance
- Please check this [internal link](https://docs.google.com/spreadsheets/d/1jkadazpbA2BUYEUdVV8Rpe54-snH1cbdJbbHsuK04-U/edit?usp=sharing) for more detailed comparison

- Overall:
  - `base/1.6` consistently outperforms `base/1.5` across most datasets and object classes
  - The largest mAP gain is seen in the `LargeBus` dataset (`+5.3 mAP`), followed by modest improvements in `JPNTaxi` and `J6Gen2`
  - `Bus` and `Truck` detection benefit most from the upgrade to `base/1.6`
  
- Dataset-level:
  - LargeBus
    - `base/1.6` shows significant improvements in truck (`+28.8`) and bicycle (`+13.0`) detection
    - Bus AP remains consistently high (`>98%`) for both, suggesting we need to add more buses in the test set 

  - J6Gen2
    - Small overall mAP improvement: `+1.2`
    - Car (`+0.7`), truck (`+2.9`), and pedestrian (`+1.2`) all show slight improvements
    - Bicycle detection remains poor, possibly due to low sample size (only 8)

  - JPNTaxi
    - Moderate mAP gain of `+2.8`
    - Strong improvements in bicycle (`+6.5`) and pedestrian (`+1.9`) detection indicate better handling of small objects.
    - Car and bus show small but meaningful gains

  - J6
    - `base/1.5` slightly edges out `base/1.6` in overall mAP (`67.1` vs `67.0`), driven by slightly better truck, bus, and bicycle APs
    - Indicates that in some mixed scenarios, `base/1.5` remains competitive

## Test performed

- Log
Tested internally and logs can be found here:
```
https://drive.google.com/drive/folders/139ZTRkHOO1-O4Y-wp9CB5Ojqa5JNP6nN?usp=drive_link
```

### CODE REVIEW

**Review by yamsam — APPROVED** [review]
LGTM
